trlx/pipeline/offline_pipeline.py

```diff
@@ -64,9 +64,17 @@ def tokenize_dialogue(  # noqa: C901
         truncated = [DialogMessage(is_output=m.is_output, tokens=m.tokens[::-1]) for m in truncated[::-1]]
 
     # remove empty messages
-    truncated = [t for t in truncated if len(t.tokens) > 0]
+    out = [t for t in truncated if len(t.tokens) > 0]
 
-    return truncated
+    if out[0].is_output:
```

Comment:
it won't, because of enforcing `<eos>` here, output is always non empty
https://github.com/CarperAI/trlx/blob/97f1c4019d67908e55f106796ce8b87df6c75e0d/trlx/pipeline/offline_pipeline.py#L42-L43